app/build.gradle

```diff
@@ -109,33 +108,30 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:17.1.0'
 
     // Serialization
-    implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.google.code.gson:gson:2.9.0'
 
     // Dependency Injection
     implementation "com.google.dagger:hilt-android:$hilt_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$composeVersion"
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
     implementation 'androidx.hilt:hilt-work:1.0.0'
     androidTestImplementation "androidx.compose.ui:ui-test-junit4:$composeVersion"
     kapt "com.google.dagger:hilt-android-compiler:$hilt_version"
-    implementation 'androidx.hilt:hilt-lifecycle-viewmodel:1.0.0-alpha03'
     kapt 'androidx.hilt:hilt-compiler:1.0.0'
 
-
     // Database
-    implementation "androidx.room:room-runtime:2.2.5"
-    kapt "androidx.room:room-compiler:2.2.5"
-    implementation 'com.facebook.stetho:stetho:1.5.1'
```

Comment:
Did you decide this wasn't needed or was it broken? Just curious. :)

Reply:
I think there are better alternatives to Stetho these days, most things you can see with just Android Studio features.

Reply:
It wasn't broken... I added a few "cleanup" things to the PR